policyengine_us/variables/gov/states/ga/tax/adjusted_gross_income/ga_agi_exclusion.py

```diff
@@ -0,0 +1,114 @@
+from policyengine_us.model_api import *
+
+
+class ga_exclusion(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Georgia retirement exclusion"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://apps.dor.ga.gov/FillableForms/PDFViewer/Index?form=2022GA500"
+        "https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=fb5db531-a80f-4790-bddb-eefc8327ef60&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A65D2-CDH3-CGX8-044N-00008-00&pdcontentcomponentid=234186&pdteaserkey=sr1&pditab=allpods&ecomp=8s65kkk&earg=sr1&prid=66f02b0a-c5ae-4162-9535-127751546807"
+    )
+    defined_for = StateCode.GA
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.ga.tax.income.agi.exclusions
+        filing_status = tax_unit("filing_status", period)
+        status = filing_status.possible_values
+        earned_income = person("earned_income", period)
+        earned_income_exclusion = min_(
+            p.retirement.cap.earned_income, earned_income
+        )
+        retirement_income = (
+            person("pension_income", period) + earned_income_exclusion
```

Comment:
need to create a ga_retirement_income variable with all the sources from [(E) (i) ](https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=a07939c6-afa6-419b-9e7d-766fc5748194&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2fshared%2fdocument%2fstatutes-legislation%2furn%3acontentItem%3a65D2-CDH3-CGX8-044N-00008-00&pdcontentcomponentid=234186&pdteaserkey=sr1&pditab=allpods&ecomp=8s65kkk&earg=sr1&prid=fb5db531-a80f-4790-bddb-eefc8327ef60)